test/test_prototype_transforms_functional.py

```diff
@@ -771,7 +778,8 @@ def _compute_expected_mask(mask, angle_, translate_, scale_, shear_, center_):
                         expected_mask[i, out_y, out_x] = mask[i, in_y, in_x]
         return expected_mask.to(mask.device)
 
-    for mask in make_segmentation_masks(extra_dims=((), (4,))):
+    # FIXME: `_compute_expected_mask` currently only works for "detection" masks. Extend it for "segmentation" masks.
+    for mask in make_detection_masks(extra_dims=((), (4,))):
```

Comment:
This is due to the `assert mask.ndim == 3`.  This was introduced in #6546 to make sure only detection masks are computed here. @vfdev-5 Could you fix the expected mask computation so both types of masks are supported? If that is done, we should use 

```suggestion
    for mask in make_detection_and_segmentation_masks(extra_dims=((), (4,))):
```

here.

This comment also applies to other occurrences of the same `FIXME` comment.